README.md

```diff
@@ -1,10 +1,12 @@
-Razor Tooling
+Razor Tooling for Visual Studio 2015
 =====
 
 AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/ks1xkw9q65ha47ip/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/razortooling/branch/dev)
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/RazorTooling.svg?branch=dev)](https://travis-ci.org/aspnet/RazorTooling)
 
-The Razor syntax provide a fast, terse, clean and lightweight way to combine server code with HTML to create dynamic web content. This repo contains tooling that interacts with the DNX Design Time Host to provide a Razor editing experience.
+## For the current Razor tooling for Visual Studio 2017, go to https://github.com/aspnet/Razor/.
+
```

Comment:
Might be worth adding a bullet about the Razor language service. After all, this repo was how users got TagHelper IntelliSense.

Could also add that information to the aspnet/Razor repo though. Maybe it makes more sense there. Leaving up to you 👍